pkg/cmd/server/server.go

```diff
@@ -455,6 +455,7 @@ func (s *server) validateBackupStorageLocations() error {
 // - Custom Resource Definitions come before Custom Resource so that they can be
```

Comment:
let's move the comment on CRDs to the top so it remains consistent with the order of the slice